packages/shopify-codemod/test/fixtures/coffeescript-soak-to-condition/assignment.output.js

```diff
@@ -0,0 +1,16 @@
+if (typeof bar === 'function' && bar() != null && bar().baz.qux != null) {
```

Comment:
This code is not the same as the previous one, as `bar()` is executed multiple times